docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #74.        else:
</pre><p> </p><p>To reduce the cognitive load of developer, we may remove the <code>else</code> part, and simply do:</p><pre class="ql-syntax" spellcheck="false">bfs = max(bfs, nn_distance)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Wait, are you doing a <code>for-else</code>? If so, let's avoid that as it will confuse people  </p>

Reply:
Yes. It is the for-else approach... and as you said... it is not easy to read. Going to change it.